README.md

```diff
@@ -2,18 +2,37 @@
 
 Query tool for SDWIS (Safe Drinking Water Information System)
 
-This package was developed as an entry to the CA Water Data Challenge 2020.  It demonstrates the utility of easy to use API wrappers to empower analysts to get more done with data with less effort. Currently the SDWIS database is inconvenient  to query and there is no public facing API. This wrapper grabs data hosted in an AWS s3 bucket so we can pretend there is already the infrastructure to support this type of development. 
+First, determine the PSIDs of interest.
 
-## Installation
+## Query metadata
```

Comment:
I don't want to merge your README as it stands into v0.1, it undos the edits Emanuel and I did.

Reply:
opps. THANK you for catching this. The readme in pkg-objects and v0.1 branches are now identical.